### ISSUE TITLE
[TYPO] LocalesApi.accountLocalesRaw()

### ISSUE THREAD

**Issue by Paul7Peterson** [phrase/phrase-js#12]
There's a typo in the line 137 of the [`LocalesApi`](https://github.com/phrase/phrase-js/blob/master/src/apis/LocalesApi.ts) file

Where "`account_id`" should be "`id`" since it's the one defined in the parameters.

```ts
const response = await this.request({
      path: `/accounts/{account_id}/locales`.replace(`{${"id"}}`, encodeURIComponent(String(requestParameters.id))), // HERE
      method: 'GET',
      headers: headerParameters,
      query: queryParameters,
  });
```

Since you are using the OpenAPI Generator, the error must come from the definitions of the backend for the interface of the arguments, most likely.

**Comment by theSoenke** [comment]
Hi @Paul7Peterson, thanks for reporting this. This looks indeed incorrect and I'll try to adapt this

**Comment by theSoenke** [comment]
I've just released version 1.5.2, which should fix this
